### DISCUSSION TITLE
I can't find copilot extension

### DISCUSSION THREAD

**Discussion by ahmedmohamed7334** [community/community#17523] [Copilot Conversations] [answered]
even I have  visual studio2022 with the required account on it but I can't find the extension in marketplace, any help?
![bandicam 2022-05-31 00-25-33-537](https://user-images.githubusercontent.com/85694013/171064662-1f203fd6-9c4e-4880-8056-5ade6b271093.jpg)

**Comment by PythonCoderAS** [comment, marked as answer]
Did you follow https://github.com/github/copilot-docs/blob/main/docs/visualstudio/gettingstarted.md#1-installing-the-visual-studio-extension? Maybe capitalization matters.

**Reply by ahmedmohamed7334** [reply]
yeah it mattered somehow thanks

**Reply by PythonCoderAS** [reply]
> yeah it mattered somehow thanks

Honestly that is very dumb. Microsoft should fix it in Visual Studio because case-sensitive search makes no sense for a marketplace

**Comment by SeanBannister** [comment]
For others who find this via search Copilot also requires Visual Studio 17.2 (they dropped support for 17.1). And won't show if you don't have a compatible version.

Capitalization had no impact on my search.
